pkg/config/system_probe_cws_windows.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Can you add this file and the `notwin` version to the CODEOWNERS file (owned by our team) ? So that it matches the rest of our config

Reply:
yup.  And renamed the confusing named `windows_notwin.go` file too